hack/test-e2e.sh

```diff
@@ -26,10 +26,7 @@ function run_e2e_tests() {
     ginkgo -v -progress \
         -focus "\[Canary\]" \
         test/e2e \
-        -- \
-        --kubeconfig="${KUBECONFIG}" \
-        --cloud-config="${CLOUDCONFIG}" \
-        --delete-namespace=false
+        -- --kubeconfig=${KUBECONFIG} --cloud-config=${CLOUDCONFIG} --delete-namespace=true
```

Comment:
Agh, the change in the Makefile above might have been to match this usage in the tests. I can still roll it back.

Reply:
It seems irrelevant to this PR so would prefer it to be reversed to simplify review.

Reply:
I rolled this back, but reckon we should have kept it. No-one will fix it in a single PR and in any other PR the same argument can be made! Anyway, should be reverted now!